docker/pypi/wmagent/init.sh

```diff
@@ -546,6 +577,10 @@ agent_upload_config(){
         elif [[ "$TEAMNAME" == *dev* ]]; then
             echo "$FUNCNAME: Dev agent, setting MaxRetries to 0..."
             agentExtraConfig='{"MaxRetries":0}'
+        elif [[ "$TEAMNAME" == Tier0* ]]; then
```

Comment:
I would move this elif statement to the second position (2nd to be evaluated), given that the other 2 could be part of a Tier0 agent setup.

Reply:
I do not think so. The Tier0's team names should always start with the Tier0*, meaning there is no inter mixture of all those options. Actually the better construction to implement those checks here would be a `case ` statement, rather than an `elif` sequence.

Reply:
Either one is fine with me. However, I still insist that matching against Tier0* is much more restrict than the dev and testbed. To avoid any "human mistake" in the future like "team=Tier0-dev1", please move it up.

Reply:
done